pom.client.xml

```diff
@@ -164,12 +164,13 @@
           </dependency>
         </dependencies>
         <configuration>
-          <effort>Max</effort>
+          <effort>min</effort>
```

Comment:
Maybe we should run nightly builds with `max` or `more` and publish the [results here](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/spotbugsXml.html#{%22tab%22:%22summary%22,%22includeFixed%22:false,%22release%22:-1,%22priority%22:4})

Min does very little and may not catch some issues.
https://spotbugs.readthedocs.io/en/stable/effort.html

Reply:
My expectation is even at min we should get everything we want.

Reply:
But I should add, there was almost no difference between min and max on my machine. The only win was from not forking the JVM for each module, so I am also happy to return this to max.